apps/website/screens/components/tag/TagPageLayout.tsx

```diff
@@ -21,6 +22,18 @@ const TagPageHeading = ({ children }: { children: React.ReactNode }) => {
             and multiple tags can be concatenated to generate a series of resources displayed with a visual hierarchy
             that calls the user&#39;s attention.
           </DxcParagraph>
+          <DxcAlert type="warning" size="fillParent">
+            This component will be removed from Halstack Design System in the next major release. Please, start
```

Comment:
I think this looks a bit better:

Please, start considering alternatives such as the `Badge` or `Chip` components.

Reply:
Done!